keyboards/checkerboards/quark_lp/config.h

```diff
@@ -46,19 +46,11 @@
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
-// ws2812 options
+// per-key RGB options
 #define RGB_DI_PIN C2 // pin the DI on the ws2812 is hooked-up to
-#define RGBLIGHT_EFFECT_BREATHING
-#define RGBLIGHT_EFFECT_RAINBOW_MOOD
-#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
-#define RGBLIGHT_EFFECT_SNAKE
-#define RGBLIGHT_EFFECT_KNIGHT
-#define RGBLIGHT_EFFECT_CHRISTMAS
-#define RGBLIGHT_EFFECT_STATIC_GRADIENT
-#define RGBLIGHT_EFFECT_RGB_TEST
-#define RGBLIGHT_EFFECT_ALTERNATING
-#define RGBLIGHT_EFFECT_TWINKLE
-#define RGBLED_NUM 14 // number of LEDs
+#define RGBLIGHT_ANIMATIONS // run RGB animations
```

Comment:
please revert this change. 

We want to explicitly enable modes rather than use a blanket enable.

Reply:
understood and done

Reply:
thanks!